HPXMLtoOpenStudio/resources/hpxml_schematron/EPvalidator.xml

```diff
@@ -1167,6 +1181,7 @@
       <sch:assert role='ERROR' test='count(h:FractionCoolLoadServed) = 1'>Expected 1 element(s) for xpath: FractionCoolLoadServed</sch:assert>
       <sch:assert role='ERROR' test='count(h:AnnualCoolingEfficiency[h:Units="EER"]/h:Value) = 1'>Expected 1 element(s) for xpath: AnnualCoolingEfficiency[Units="EER"]/Value</sch:assert>
       <sch:assert role='ERROR' test='count(h:AnnualHeatingEfficiency[h:Units="COP"]/h:Value) = 1'>Expected 1 element(s) for xpath: AnnualHeatingEfficiency[Units="COP"]/Value</sch:assert>
+      <sch:assert role='ERROR' test='count(h:AttachedToGeothermalLoop) &lt;= 1'>Expected 0 or 1 element(s) for xpath: AttachedToGeothermalLoop</sch:assert>
```

Comment:
Since this element is optional, it is possible for someone to write an HPXML file where they include a GSHP and a geothermal loop, but don't connect them via this element. If that were to happen, the geothermal loop inputs would be ignored and all defaults would be used, which is not a good outcome.

This is a similar situation to what can arise for HVAC distribution systems. So you can follow that example and add an error-check like [this](https://github.com/NREL/OpenStudio-HPXML/blob/4fd2701e4f70734a17dd7bd3306f351d1f52e19f/HPXMLtoOpenStudio/resources/hpxml.rb#L4488) and a test like [this](https://github.com/NREL/OpenStudio-HPXML/blob/4fd2701e4f70734a17dd7bd3306f351d1f52e19f/HPXMLtoOpenStudio/tests/test_validation.rb#L870).